### ISSUE TITLE
Fresh Add of MDSVEX Fails

### ISSUE THREAD

**Issue by BryanSchuetz** [svelte-add/mdsvex#10]
On a fresh install:

`npm init svelte@next`
`npm install`
`npx svelte add mdsvex`
`npm install`
`npm run dev`

The following error is returned when requesting the `/example-markdown` route: "Failed to parse source for import analysis because the content contains invalid JS syntax. You may need to install appropriate plugins to handle the .md file format."

**Comment by babichjacob** [comment]
This is because of the CJS -> ESM config file change in SvelteKit and I am currently working on fix it.

**Comment by babichjacob** [comment]
closed by 4eef4e6645b110b8785b52fff164f640b8ef4c2b

**Comment by BryanSchuetz** [comment]
FWIW, I still have problems with this on fresh install: 

```
Malformed svelte.config.js
file:///Users/bryanschuetz/Projects/distess-signal/svelte.config.js:1
import { mdsvex } from "mdsvex";
         ^^^^^^
SyntaxError: The requested module 'mdsvex' is expected to be of type CommonJS, which does not support named exports. CommonJS modules can be imported by importing the default export.```

Making the suggested change seems to make it all work for me: 

import pkg from 'mdsvex';
const { mdsvex } = pkg;

**Comment by babichjacob** [comment]
Thanks for the report. What version of Node are you on? I only tested 16 and I didn't need this.

**Comment by BryanSchuetz** [comment]
14.4

**Comment by babichjacob** [comment]
Right, that's outdated. I was able to reproduce on 14.4 but not newer versions like 14.16.1 or the aforementioned 16.1.0.

**Comment by babichjacob** [comment]
Just learned in the Discord server you'd run into other errors in your SvelteKit app [for the same reason](https://discord.com/channels/457912077277855764/819723698415599626/839243703007838208), so this isn't me enforcing a requirement that regular SvelteKit doesn't have 😉.

**Comment by babichjacob** [comment]
Let me know if that works! Thanks!

**Comment by BryanSchuetz** [comment]
That wasn't me—but, FWIW after updating node, ESLint still screams at me constantly about this.
<img width="529" alt="Screen Shot 2021-05-05 at 1 55 24 PM" src="https://user-images.githubusercontent.com/174269/117187351-d20bd080-ada9-11eb-9794-541ce1eb2f1a.png">
